tests/modules/bbmap/pileup/test.yml

```diff
@@ -0,0 +1,10 @@
+- name: "bbmap pileup"
+  command: nextflow run ./tests/modules/bbmap/pileup -entry test_bbmap_pileup -c ./tests/config/nextflow.config -c ./tests/modules/bbmap/pileup/nextflow.config
+  tags:
+    - "bbmap"
+    - "bbmap/pileup"
+  files:
+    - path: "output/bbmap/test.bam"
```

Comment:
Will need to be updated to test for stats and hist outputs

Reply:
Good catch, got derailed when the CI was down this morning!